### PR TITLE
restore ignore field from deletedAt attribute

### PR DESCRIPTION
**Pull request by sushantdhiman**
Please check #6805

### CODE REVIEW

**Comment by mickhansen** [comment]
When was this changed?

**Comment by sushantdhiman** [comment]
> When was this changed?

? @mickhansen , I dont know when this was changed :D

**Comment by mickhansen** [comment]
Oh my bad, i read `restore` differently.

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Future
+- [FIXED] `restore` don't use `field` from `deletedAt`
```
Comment (mickhansen) [inline changelog.md]: `now uses`

**Pull request merged**